docs/support-tasks.md

```diff
@@ -110,12 +110,30 @@ Depending on the number of emails to send, the Rake task can take a few minutes
 bundle exec rake 'support:resend_failed_emails:by_id[<email_one_id>,<email_two_id>]'
 ```
 
-## Get subscriber numbers for a list
+## Get subscriber list csv report
 
-To see the number of subscribers for a given list:
+To see a csv report for the number of subscribers for a given list:
 
 ```bash
 bundle exec rake 'report:csv_subscriber_lists[<on_date>] SLUGS=<list_slug>'
 ```
 
 The date should be in ISO8601 format, for example 2020-01-01.
+
+## Get a simple count of subscribers to a list by its URL
+
+To see a simple count of the number of subscribers for a given list:
+
+```bash
+bundle exec rake 'report:csv_subscriber_lists[<url>]'
+```
+This will always report on the number of active subscriptions as of now.
+
+You can also pass it a :active_on_datetime which will count how many active subscriptions there were on a particuar date.
+(active is defined as created after that datetime and not with an "ended_one" datetime by the given date)
```

Comment:
Should this be `before`? (also you have a typo)

```suggestion
(active is defined as created before that datetime and not with an "ended_on" datetime by the given date)
```

Reply:
yep! thanks!

I changed the implimentation of this and missed these docs